iib/workers/tasks/dc_utils.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from iib.workers.tasks.utils import skopeo_inspect
+
+
+def is_image_dc(image):
+    """
+    Detect declarative config image.
+
+    We can have two types of image - SQLite and Declarative Config
+    Those can be distinguished by LABELS.
+    Declarative config will have defined this LABEL: operators.operatorframework.io.index.configs.v1
+
+    :param str image: the pull specification of the container image (usually from_image)
+    :return: True if image is declarative config type, False otherwise
+    :rtype: bool
+    """
+    skopeo_output = skopeo_inspect(f'docker://{image}')
+    dc_image_label = 'operators.operatorframework.io.index.configs.v1'
```

Comment:
can you move this to global scope?

Reply:
I can, but can U ask why is it needed?
You want to have it easily accessible? However for what when you can use this function to test presence of this label for you.

Reply:
Any idea if there would be any future cases where we will have to set/unset the values for this label?

Reply:
This value is set during migration from SQLite to FBC (declarative config). 
However it is done automatically when we generate Dockerfile using `opm alpha generate dockerfile`. So it is not directly in our code.

Reply:
ack, Thanks